pandas/_libs/tslibs/strptime.pyx

```diff
@@ -511,14 +547,17 @@ class TimeRE(dict):
             # The " \d" part of the regex is to make %c from ANSI C work
             'd': r"(?P<d>3[0-1]|[1-2]\d|0[1-9]|[1-9]| [1-9])",
             'f': r"(?P<f>[0-9]{1,9})",
+            'G': r"(?P<G>\d\d\d\d)",
```

Comment:
I am not sure we have a doc-refernce to the strptime codes anywhere @mroeschke do you know?

Reply:
Looks like the reference is here: https://github.com/pandas-dev/pandas/blob/fcc11feb1caf04061195d1b21c4952a4c31fa23a/pandas/_libs/tslibs/strptime.pyx#L361

Reply:
right so we have a doc reference anywhere though? eg in a doc string for Timestamp / to_datetime and / or in the docs proper?

Reply:
Yeah looks like the we link to the directives here: https://pandas.pydata.org/pandas-docs/stable/user_guide/timeseries.html#providing-a-format-argument